### DISCUSSION TITLE
I have selected a theme for Rofi in the Rofi theme selector and now I cannot return the original theme.

### DISCUSSION THREAD

**Discussion by Dolbaeb777K** [JaKooLit/Hyprland-Dots#167] [Q&A] [answered]
How do I choose the original theme? I don't find her on the list..

**Comment by JaKooLit** [comment, marked as answer]
I have seen this before

on the last part of ~/.config/rofi/config.rasi

delete the last line which is referring to @theme....something

**Reply by Dolbaeb777K** [reply]
Thanks very much!
